Update submodules with excerpter binary files fix

parlough opened the pull request:
_Description of what this PR is changing or adding, and why:_ Updates the codelabs and site_shared submodule with the new fix for skipping binary files.

_Issues fixed by this PR (if any):_ [N/A](https://github.com/flutter/website/issues/7001)

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

domesticmouse commented on src/get-started/codelab.md:
```diff
@@ -613,57 +607,49 @@ lazily, on demand.
        Widget build(BuildContext context) {
     -    final wordPair = WordPair.random();
     -    return Text(wordPair.asPascalCase);
-    +    return Scaffold(
```
Because I delayed moving the Scaffold until a later step of the codelab, there probably needs to be an update to the surrounding codelab text to reflect that this change.

parlough replied:
Taking a look now :)

parlough replied:
Step 5 (the last one of this part) is a bit empty now, only changing the title of the app bar.

I think for now, this is fine, but perhaps followup work should consider moving that an earlier step?

domesticmouse replied:
Sounds good

domesticmouse commented:
PTAL @sfshaza2 =)

parlough commented:
I just noticed the text was duplicating some other instructions, even before the codelab updates. I'm going to convert this to a draft temporarily while I address those problems.

sfshaza2 commented:
Thanks for tackling this, @parlough!!!

parlough commented:
PTAL @sfshaza2 - I had to make some textual changes due to changes in the Get Started codelab code. As I was making those adjustments, I noticed the instructions weren't updated properly for other previous changes either, leaving some steps duplicated or out of date. 

I modified some old steps and introduced a new one to try to make things more broken up. Let me know if you'd like to see any changes. Thanks!

sfshaza2 reviewed (approved):
@parlough, whatever we pay you, it's not enough.  :D. The text changes all look great! Many thanks!!!
